homeassistant/components/roomba/__init__.py

```diff
@@ -26,19 +29,33 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-CONFIG_SCHEMA = vol.Schema(
+
+def _has_all_unique_prefixes(value):
+    """Validate that each vacuum configured has a unique prefix.
+
+    Uniqueness is determined case-independently.
+    """
+    prefixes = [device[CONF_PREFIX] for device in value]
```

Comment:
```suggestion
    prefixes = [device[CONF_BLID] for device in value]
```

Reply:
Might make sense to check CONF_HOST instead actually now that I think about it.

Which would make it `_has_all_unique_hosts`